app/api/rest_api.py

```diff
@@ -68,9 +70,15 @@ async def landing(self, request):
     async def rest_core(self, request):
         try:
             access = dict(access=tuple(await self.auth_svc.get_permissions(request)))
-            data = dict(await request.json())
+            if request.method == 'GET':
+                data = dict(index='default', path=request.path[10:])
```

Comment:
I think you can do `request.match_info['tail']` here instead of `request.path[10:]`

Reply:
yea, that makes sense, let me try that out